README.md

```diff
@@ -29,7 +29,7 @@ her what we need. Current info (as of 2014 June 11):
 - Fax:   604-464-5056
 - Email: cneff@coca-cola.com
 
-As she is often not in the office, it's best to call her cell.
+As Chantelle is often not in the office, it is best to call her cell (**604-830-4022**).
```

Comment:
Well, the cell number is right above. Isn't it redundant to have it here?

Reply:
I am not sure if people still use this code but the 376 assignment wants us to improve the clarity of documents.
HMMMMMM. I put it that to emphasized the importance of the number.
We can just preserve the pronoun edit.